easybuild/easyblocks/__init__.py

```diff
@@ -43,7 +43,7 @@
 # recent setuptools versions will *TRANSFORM* something like 'X.Y.Zdev' into 'X.Y.Z.dev0', with a warning like
 #   UserWarning: Normalizing '2.4.0dev' to '2.4.0.dev0'
 # This causes problems further up the dependency chain...
-VERSION = LooseVersion('4.1.2.dev0')
+VERSION = LooseVersion('4.2.0.dev0')
```

Comment:
So, just checking if this is correct; should it be 4.2.0.dev0 here?

Reply:
Ah, no, should be `4.2.1.dev0`, of course...

Reply:
fixed with force push in e7e45bc44